docs/nodes/build/set-up-node-with-installer.md

```diff
@@ -13,17 +13,16 @@ installs](#unattended-installation).
 ## Before You Start
 
 Avalanche is an incredibly lightweight protocol, so nodes can run on commodity
-hardware with the following minimum specifications. Note that as network usage
-increases, hardware requirements may change.
+hardware with the following minimum specifications. 
 
 - CPU: Equivalent of 8 AWS vCPU
 - RAM: 16 GiB
 - Storage: 1 TiB
 - OS: Ubuntu 20.04 or MacOS &gt;= 12
 - Network: sustained 5Mbps up/down bandwidth
 
-Please note that HW requirements shall scale with the amount of AVAX staked on
-the node. Nodes with big stakes (100k+ AVAX) will need more powerful machines
+NOTE: HW requirements shall scale with the amount of AVAX staked on
```

Comment:
Should docusaurus admonition https://docusaurus.io/docs/markdown-features/admonitions be used here?